src/server/index.jsx

```diff
@@ -165,41 +162,38 @@ server
       });
     },
   )
-  .get(
-    [articleRegexPath, frontpageRegexPath, ...mediaRadioAndTvRegexPathsArray],
-    async ({ url, headers, path: urlPath }, res) => {
+  .get('/*', async ({ url, headers, path: urlPath }, res) => {
+    try {
+      const { service, isAmp, route, match } = getRouteProps(routes, url);
+      const data = await route.getInitialData(match.params);
+      const { status } = data;
+      const bbcOrigin = headers['bbc-origin'];
+
+      let dials = {};
```

Comment:
I wonder if you could do

```js
const dials = await getDials().catch(message => {
  logger.error(`Error fetching Cosmos dials: ${message}`);
  return {};
});
```

instead of creating an object and then potentially mutating it

Reply:
I havent touched this, the git diff is just weird 🙈 https://github.com/bbc/simorgh/pull/3307 removes this code anyway, so gunna leave it for now :)